src/Components/BankDebitForm/__tests__/BankDebitForm.jest.tsx

```diff
@@ -4,6 +4,7 @@ import { useSystemContext } from "System"
 import { useTracking } from "react-tracking"
 import { PaymentTestQueryRawResponse } from "__generated__/PaymentTestQuery.graphql"
 import { BankDebitForm } from "../BankDebitForm"
+import { useOrderPaymentContext } from "../../../Apps/Order/Routes/Payment/PaymentContext/OrderPaymentContext"
```

Comment:
`"Apps/Order/Routes/Payment/PaymentContext/OrderPaymentContext"` should just work here and below